packages/legacy/core/App/container-impl.ts

```diff
@@ -97,19 +106,20 @@ export class MainContainer implements Container {
         tours: { ...defaultState.tours, ...tours },
         onboarding: { ...defaultState.onboarding, ...onboarding },
       }
+
       dispatch({ type: DispatchAction.STATE_DISPATCH, payload: [state] })
     })
 
     return this
   }
 
   public resolve<K extends keyof TokenMapping>(token: K): TokenMapping[K] {
-    return this.container.resolve(token) as TokenMapping[K]
+    return this._container.resolve(token) as TokenMapping[K]
   }
 
-  public getContainer(): DependencyContainer {
-    return this.container
-  }
+  // public resolveSome<K extends keyof TokenMapping>(tokens: K[]): TokenMapping[K][] {
+  //   return tokens.map((token) => this.resolve(token))
+  // }
```

Comment:
Does this work? Is it meant to be commented out or should it be removed?

Reply:
It was what I hoped work but it does not. It makes the TS type-checker sad. Will remove and save locally in case I feel like trying again at a later date.